File: C/TH1/bai2.c
#include <math.h>
#include <stdio.h>

int main() {
  int a, b, c;
  printf("Nhap vao canh a: ");
  scanf_s("%d", &a);
  printf("Nhap vao canh b: ");
  scanf_s("%d", &b);
  printf("Nhap vao canh c: ");
  scanf_s("%d", &c);

  if (a + b < c || a + c < b || c + b < a)
    printf("khong phai la 3 canh tam giac");
  else if (a == b && b == c) {
    printf("day la tam giac deu");
  } else if (a == b || a == c || b == c) {
    printf("day la tam giac can");
  } else {
    printf("day la tam giac thuong");
  }
}

File: Python/school/DangHuyPhong_66KTPM2_BTVN/buoi_8/bai2.py
a = input("Nhap vao canh a: ")
b = input("Nhap vao canh b: ")
c = input("Nhap vao canh c: ")

try:
    a = float(a)
    b = float(b)
    c = float(c)
    if a <= 0 or b <= 0 or c <= 0:
        raise ValueError("Mot trong nhung canh a,b,c nho hon bang 0")
    if a + b <= c or a + c <= b or b + c <= a:
        raise ValueError("a,b,c khong the la canh cua tam giac")
    print("a,b,c hop le")

except ValueError as ve:
    print(ve)
